.storybook/main.js

```diff
@@ -59,7 +60,9 @@ module.exports = {
           {
             loader: 'postcss-loader',
             options: {
+              ident: 'postcss',
```

Comment:
Documentation says this `ident` line is needed. If anyone wants a reference, I can add, but thought it might be ok to leave off.